### ISSUE TITLE
laser scan

### ISSUE THREAD

**Issue by mustafasamed** [gstavrinos/jaguar#7]
Hello,
  I want to navigate my jaguar. Yet I need to make a map and there is no laser scan for /scan topic. Even though there is no laser scan, you have made a place for it. Do I have to erase your created place for laser scan? and if I don't need to erase it, what should I do in order to put laser scan in your laser place. Thank you for understanding. Have a nice day.

**Comment by gstavrinos** [comment]
Check out this package that contains drivers for Jaguar's motor controller, laser scanner etc:

https://github.com/gstavrinos/jaguar_robot

**Comment by mustafasamed** [comment]
I checked it, yet it lacks jaguar4x4/config/laser_config.yaml there is no file named jaguar4x4 and I could not find the yaml in google/github

**Comment by gstavrinos** [comment]
You are right, there is a problem there. I will look into it. For now, just comment out the laser filters part. Keep in mind though, that you should be using a launch file that is specific to your laser scanner model.

**Comment by mustafasamed** [comment]
thanks. By the way to develop these files. There is one more problem, which is about ekf_localization. When I launch rviz, wheels of jaguar destabilize. When I searched the cause of it, I found there is too many publishers of tf about wheel. Then I killed ekf_localization and problem solved. However, you must not put it without a reason that's why I want to ask you can I continue navigation without ekf_localization? if not, how can I solve the problem? Thanks for your efforts by the way :-)
